assets/js/main.js

```diff
@@ -139,14 +139,9 @@ $('.platform-picker li').click(e => {
 [['event'], ['acolytes'], ['cetus'], ['earth'], ['bounties'], ['alerts'],
   ['news'], ['invasions'], ['reset'], ['sortie'], ['fissures'],
   ['baro'], ['darvo'], ['deals', 'false']].forEach(([component, defValue]) => {
-  let value = Cookies.get(component);
-  if (typeof value === 'undefined') {
-    if (typeof defValue === 'undefined') {
-      value = 'true';
-    } else {
-      value = defValue;
-    }
-    Cookies.set(component, value, {expires: 365});
+  const value = localStorage.getItem(component);
+  if (typeof value === 'undefined' || value === null) {
+    localStorage.setItem(component, defValue || 'true');
```

Comment:
Not relevant to the review, but is there a reason for using the string literals instead of booleans?